packages/admin/docs/02-resources/06-relation-managers.md

```diff
@@ -395,6 +395,16 @@ In this example, `$action->getRecordSelect()` outputs the select field to pick t
 
 Please ensure that any pivot attributes are listed in the `withPivot()` method of the relationship *and* inverse relationship.
 
+### Scoping attachable models
+
+You may want to apply a scope to the models available for attaching in a relation manager:
+
+```php
+use Filament\Tables\Actions\AttachAction;
+
+AttachAction::make()->recordSelectOptionsQuery(fn ($query) => $query->whereBelongsTo(auth()->user())
```

Comment:
Please add the `recordSelectOptionsQuery()` method on a new line in both examples, and `$query` should be an instance of `Builder`, and that class needs importing too

Reply:
All set Dan.